### PR TITLE
feat: support recycle bin in content navigation #152

### PR DESCRIPTION
**Pull request by boyce-w**
Summary
Support Recycle Bin in the SAS Content Navigation Pane.
See details in the issue https://github.com/sassoftware/vscode-sas-extension/issues/152

Testing
How did you test this change?
Test manually and run all the unit tests

### CODE REVIEW

**Comment by boyce-w** [comment]
Hi @scottdover @scnwwu @Zhirong2022 , I migrate the PR #176 to this new one for some reason. Please help continue on testing and reviewing here. Thanks.

**Comment by Zhirong2022** [comment]
The feature works as expected.

**Review by Zhirong2022 — APPROVED** [review]

File: client/src/components/ContentNavigator/const.ts [hunk 1]
```diff
@@ -38,17 +38,24 @@ export const FOLDER_TYPES = [
...
+  FileDeletionError: "Unable to delete file.",
+  FolderDeletionError: "Unable to delete folder.",
+  DeletionWarningMessage:
+    'Are you sure you want to permanently delete the item "{name}"?',
+  DeleteButtonLabel: "Delete",
+  FileRestoreError: "Unable to restore file.",
+  FolderRestoreError: "Unable to restore folder.",
+  EmptyRecycleBinWarningMessage:
+    "Are you sure you want to permanently delete all the items? You cannot undo this action.",
+  EmptyRecycleBinError: "Unable to empty the recycle bin.",
```
Comment (scottdover) [inline client/src/components/ContentNavigator/const.ts:58]: Can you alphabetize these with the remaining items, please?

File: package.nls.json [hunk 2]
```diff
@@ -29,6 +29,8 @@
   "commands.SAS.deleteResource": "Delete",
   "commands.SAS.refreshResources": "Refresh",
   "commands.SAS.renameResource": "Rename...",
+  "commands.SAS.restoreResource": "Restore",
+  "commands.SAS.emptyRecycleBin": "Empty Recycle Bin",
```
Comment (scottdover) [inline package.nls.json:33]: Can you alphabetize these?

**Comment by scottdover** [comment]
It looks like you have conflicts w/ ContentModel. Other than that, this is good to go once those minor changes are taken care of

**Review by scnwwu — APPROVED** [review]

**Pull request merged**